lib/chat/chat_messenger.dart

```diff
@@ -274,25 +289,48 @@ class ChatMessengerState extends State<ChatMessenger> {
       children: <Widget>[
         Scaffold(
           appBar: AppBar(
+            // remove bottom shadow
+            elevation: 0.0,
             leading: IconButton(
               onPressed: () {
                 Navigator.pop(context);
               },
-              icon: Icon(Icons.arrow_back, color: Colors.white)
-            ),
-            backgroundColor: Theme.of(context).primaryColor,
-            centerTitle: true,
-            title: Text(
-              widget.recipient,
-              style: GoogleFonts.muli(
-                textStyle: TextStyle(
-                  color: Colors.white, 
-                  letterSpacing: .5, 
-                  fontSize: 30.0, 
-                  fontWeight: FontWeight.bold
-                )
+              icon: Icon(
+                Icons.arrow_back_ios,
+                color: Theme.of(context).accentColor
               )
             ),
+            backgroundColor: Colors.white,
+            centerTitle: true,
+            title: Column (
```

Comment:
Same here! Maybe make a separate function that returns the title widget so it's not overwhelming